notify-send.sh

```diff
@@ -61,6 +75,10 @@ convert_type() {
     esac
 }
 
+make_action() {
+  echo "\"notify-send: $2\", \"$1\""
```

Comment:
Is `notify-send` a uniquely chosen string by which we identify our actions? If so can it be `notify-send.sh` then?

Reply:
Done